actions.yaml

```diff
@@ -0,0 +1,45 @@
+list-versions:
```

Comment:
this file can now go under charmcraft.yaml `actions` section
https://juju.is/docs/sdk/charmcraft-yaml#heading--actions

Reply:
done